### ISSUE TITLE
Login- and signup-button are cutoff on Android phones with screen ratio 19.5:9

### ISSUE THREAD

**Issue by evenka** [auth0/lock#1706]
### Description

Both the login- and signup-button are cutoff and partially hidden from view. When scrolling the button is correctly rendered, but this is nonetheless a buggy experience for new users trying to create an account.

![auth0_bug](https://user-images.githubusercontent.com/51944492/61626775-7593fb80-ac7e-11e9-80c8-171c52bf8984.png)

### Environment

Please provide the following:

Lock version: 11.15

* Device: OnePlus A6013
* Browser version: Chrome 75.0.3770.143
* OS: Android 9

* Device: Huawei Mate 10
* Browser version: Chrome 75.0.3770.143
* OS: Android 9

### Reproduction

1. Navigate to development.equlitycheck.it.
2. Signup as a new user.
3. The Auth0-popup consistenly produces the bug for affected phones.

**Comment by fsc7** [comment]
I am having the same issue with samsung galaxy s6.

**Comment by fsc7** [comment]
My problem happens in Chrome, it seems that the CSS style is considering the whole viewport but Chrome is using some pixels on top with its toolbar. This should resolve the problem:
https://css-tricks.com/the-trick-to-viewport-units-on-mobile/

**Comment by madine** [comment]
Having the same issue on chrome too (pixel 1)

**Comment by fsc7** [comment]
Is it possible to reopen this issue?

**Comment by erowell** [comment]
I have this issue on a samsung note 8 using chrome as well, can this be re-opened?

**Comment by stevehobbsdev** [comment]
Yes, let me re-open and investigate.

**Comment by stevehobbsdev** [comment]
👋 I have raised #1785 which should fix this, thanks to @fsc7 for the pointer to the fix. I tried to test on as many devices in Browserstack that I could (screenshots in the PR).

**Comment by erowell** [comment]
@stevehobbsdev - Thanks for re-opening the issue and the quick turnaround on that fix!

**Comment by fsc7** [comment]
Hello,
Thanks @stevehobbsdev for the work on the issue, I just could take a look at it now. When should I see the results? In my mobile Samsung S6 I still get login page cut.
![Screenshot_20200127-102040](https://user-images.githubusercontent.com/9878072/73159545-69866c00-40ef-11ea-8ce2-e372e3c0086a.png)

**Comment by stevehobbsdev** [comment]
@fsc7 Are you using the hosted login page? We haven't released the build yet so you won't see it on live. I'm hoping to do that later today; will report back once it's out.

**Comment by fsc7** [comment]
Great, looking forward to that. Thanks.

**Comment by fsc7** [comment]
I still have the issue and I noticed that in your tests you don't have the social buttons enabled. Do you think this might be causing problem in my case?

**Comment by stevehobbsdev** [comment]
@fsc7 Even though I didn't have any screenshots for this, I'm pretty sure I tested this 🤔 let me try again and come back to you

**Comment by stevehobbsdev** [comment]
@fsc7 I have some social connections enabled and the issue is still resolved for me: 

<img width="368" alt="image" src="https://user-images.githubusercontent.com/766403/73737966-77fe0480-473b-11ea-8b40-c08f437690e2.png">

If you are still getting issues with the login button, could you please raise a new ticket with some more screenshots of what you're seeing? Just don't want this to get lost.

**Comment by fsc7** [comment]
![20200204_135232](https://user-images.githubusercontent.com/9878072/73746075-19e21900-475d-11ea-9d03-aa39b4868f4e.png)
Hello, here it goes

**Comment by fsc7** [comment]
This issues should not be closed since the problem persists.

**Comment by stevehobbsdev** [comment]
@fsc7 Can you confirm which version of Lock you are using? Are you using the default, uncustomized template or are you using a custom version?

**Comment by fsc7** [comment]
Hello @stevehobbsdev , I am sorry, my fault, I updated the lock version to the latest and it is working fine now. Many thanks.

**Comment by stevehobbsdev** [comment]
@fsc7 No problem! Glad it's fixed :)
